internal/rest/getv2clustersname_test.go

```diff
@@ -54,6 +54,7 @@ func setupMockServer(t *testing.T, expectedCluster capi.Cluster, expectedActiveP
 	machineResource.EXPECT().List(mock.Anything, mock.Anything).Return(&unstructured.UnstructuredList{Items: []unstructured.Unstructured{*machine}}, nil).Maybe()
 	nsMachineResource := k8s.NewMockNamespaceableResourceInterface(t)
 	nsMachineResource.EXPECT().Namespace(activeProjectID).Return(machineResource).Maybe()
+	nsMachineResource.EXPECT().Namespace(expectedActiveProjectID).Return(machineResource).Maybe()
```

Comment:
That's a good catch that we actually didn't use `expectedActiveProjectID` in `setupMockServer`

Instead of adding this additional line, i'd prefer to fix this method to use `expectedActiveProjectIdD` from it's arguments, not global variable `activeProjectID`

Reply:
Also this change was introduced to `setupMockServer`, but newly written test does not use it.
Is this change required?